### PR TITLE
Update ed example

### PR DESCRIPTION
**Pull request by patsy-issa**

### CODE REVIEW

File: examples/ember-data/app/models/album.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import Model from 'ember-data/model';
-import attr from 'ember-data/attr';
-import { hasMany } from 'ember-data/relationships';
+import Model, { attr, hasMany } from "@ember-data/model";
 
-export default Model.extend({
-  title:    attr('string'),
-  coverUrl: attr('string'),
+export default class Album extends Model {
+  @attr("string") title;
```
Comment (marcoow) [inline examples/ember-data/app/models/album.js:4]:
```suggestion
  @attr title;
```

File: examples/ember-data/app/models/album.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-  songs: hasMany(),
-  comments: hasMany()
-});
+  @attr("string") coverUrl;
```
Comment (marcoow) [inline examples/ember-data/app/models/album.js:6]:
```suggestion
  @attr coverUrl;
```

File: examples/ember-data/app/models/comment.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-import attr from 'ember-data/attr';
-import { belongsTo } from 'ember-data/relationships';
+import Model, { attr, belongsTo } from "@ember-data/model";
 
-export default Model.extend({
-  rating:    attr('number'),
-  text:      attr('string'),
-  createdAt: attr('date'),
+export default class Comment extends Model {
+  @attr("number") rating;
```
Comment (marcoow) [inline examples/ember-data/app/models/comment.js:4]:
```suggestion
  @attr rating;
```

File: examples/ember-data/app/models/comment.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 
-  album: belongsTo()
-});
+  @attr("string") text;
```
Comment (marcoow) [inline examples/ember-data/app/models/comment.js:6]:
```suggestion
  @attr text;
```

File: examples/ember-data/app/models/song.js [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
-import attr from 'ember-data/attr';
-import { belongsTo } from 'ember-data/relationships';
+import Model, { attr, belongsTo } from "@ember-data/model";
 
-export default Model.extend({
-  title:    attr('string'),
-  duration: attr('number'),
-  mp3Url:   attr('string'),
+export default class Song extends Model {
+  @attr("string") title;
```
Comment (marcoow) [inline examples/ember-data/app/models/song.js:4]:
```suggestion
  @attr title;
```

File: examples/ember-data/app/models/song.js [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
 
-  album: belongsTo()
-});
+  @attr("number") duration;
```
Comment (marcoow) [inline examples/ember-data/app/models/song.js:6]:
```suggestion
  @attr duration;
```

File: examples/ember-data/app/models/song.js [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
+
+  @attr("string") mp3Url;
```
Comment (marcoow) [inline examples/ember-data/app/models/song.js:8]:
```suggestion
  @attr mp3Url;
```

File: examples/ember-data/app/models/album.js (hunk 1 above)
Comment (patsy-issa) [reply examples/ember-data/app/models/album.js:4]: Any particular reason to rely on ED to automatically determine the type vs being explicit?

Comment (marcoow) [reply examples/ember-data/app/models/album.js]: I don't know the exact reason tbh but I know it's the recommendation.

**Review by marcoow — APPROVED** [review]

**Comment by marcoow** [comment]
@patsy-issa 

> No description provided.

🙏

**Pull request merged**